lldb/test/API/functionalities/asan/TestMemoryHistory.py

```diff
@@ -15,17 +15,23 @@ class AsanTestCase(TestBase):
     @expectedFailureNetBSD
     @skipUnlessAddressSanitizer
     def test(self):
-        self.build(make_targets=["asan"])
+        self.build(make_targets=["compiler_rt-asan"])
         self.asan_tests()
```

Comment:
Maybe this method should be renamed to self.compiler_rt_asan_tests ,   (self.libsanitizers_asan_tests test same functionality, different way)

Reply:
Done.  This preserves consistency.